urbit/umbrel-app.yml

```diff
@@ -1,44 +1,43 @@
-
-
-  If you are interested in purchasing an L2 planet checkout: subject.network/buy
-
-
-  Credit to ~timluc-miptev, ~master-forwex, ~sipsen-pilser & ~rivpyl-sidfyl
-developer: ~mopfel-winrux & ~sitful-hatred
-website: https://github.com/mopfel-winrux/urbit-umbrel
-dependencies: []
-repo: https://github.com/mopfel-winrux/urbit-umbrel
-support: https://github.com/mopfel-winrux/urbit-umbrel/issues
-port: 8090
-gallery:
-  - 1.jpg
-  - 2.jpg
-  - 3.jpg
-path: /launch
-defaultUsername: umbrel
-deterministicPassword: true
-torOnly: false
-submitter: ~mopfel-winrux
-submission: https://github.com/getumbrel/umbrel/pull/1246
-releaseNotes: >-
-  Updated Urbit binary to v1.16
-  
-  Arvo 417K
-  
-  Vere 1.16
-  
-  This release is functionally equivalent to v1.15 but is built in the new Vere repo using Bazel instead of Nix with a new and improved release pipeline. It marks the successful extraction of the runtime from the Urbit monorepo. It also adds a new AArch64-specific macOS build and removes the Windows build.
+manifestVersion: 1
+id: urbit
+category: Networking
+name: Urbit
+version: "v1.16-hotfix1"
+tagline: Run Urbit on your Umbrel
+description: >-
+  Urbit is a personal server for self-sovereign personal & networked
+  computing. Nock, a functional combinator, is built into Hoon, a system
+  programming language, used to implement Arvo, a deterministic operating
+  system, that communicates over Ames, a decentralized, encrypted P2P network.
+
+
+  This app helps you boot and manage Urbit instances on your Umbrel. It provides a simple management GUI, start and stop operations, and passes your +code through the interface.
+
+
+  If you are interested in purchasing an L2 planet checkout: subject.network/buy
+
+
+  Credit to ~timluc-miptev, ~master-forwex, ~sipsen-pilser & ~rivpyl-sidfyl
+developer: ~mopfel-winrux & ~sitful-hatred
+website: https://github.com/mopfel-winrux/urbit-umbrel
+dependencies: []
+repo: https://github.com/mopfel-winrux/urbit-umbrel
+support: https://github.com/mopfel-winrux/urbit-umbrel/issues
+port: 8090
+gallery:
+  - 1.jpg
+  - 2.jpg
+  - 3.jpg
+path: /launch
+defaultUsername: umbrel
+deterministicPassword: true
+torOnly: false
+submitter: ~mopfel-winrux
+submission: https://github.com/getumbrel/umbrel/pull/1246
+releaseNotes: >-
+  - Updated Urbit binary to v1.16
+
+  - Fix URL with vere repo
+
+  This release is functionally equivalent to [v1.15](https://github.com/urbit/urbit/releases/tag/urbit-v1.15) but is built in the [new Vere repo](https://github.com/urbit/vere) using [Bazel](https://bazel.build) instead of [Nix](https://nixos.org) with a new and improved release pipeline. It marks the successful extraction of the runtime from the [Urbit monorepo](https://github.com/urbit/urbit). It also adds a new AArch64-specific macOS build and removes the Windows build.

```

Comment:
Markdown is not currently supported and should be plaintext; could you change the release notes to be more friendly when this is plaintext?

Would you also know why git is marking this entire file as changed?